apps/vaporgui/MainForm.cpp

```diff
@@ -394,7 +395,13 @@ MainForm::MainForm(
 	}
 
 	if (files.size() && files[0].endsWith(".nc")) {
-        loadData(files[0].toStdString());
+        VDCNetCDF vdc;
+        int ret = vdc.Initialize(files[0].toStdString(), {}, VDC::R);
```

Comment:
Fixed #40fb014

Reply:
Because VDC::Initialize logs errors you need to disable error logging if you 're not going to report the error. E.g.

bool errEnabled = MyBase::EnableErrMsg(false);
int ret = vdc.Initialize(files[0].toStdString(), {}, VDC::R);
MyBase::EnableErrMsg(errEnabled);

Reply:
I did but it was lost. If you look at commit 40fb014 I added `MyBase::EnableErrMsg(false);` but it was somehow removed in a later commit

Reply:
I double checked and it was in fact already in there. My code was just setting it to false and true however so I updated it to save and restore whatever is set.